packages/block-library/src/navigation/edit/index.js

```diff
@@ -231,37 +188,49 @@ function Navigation( {
 		}
 	} );
 
-	if ( isPlaceholderShown ) {
-		const PlaceholderComponent = CustomPlaceholder
-			? CustomPlaceholder
-			: NavigationPlaceholder;
+	// We don't want to render a missing state if we have any inner blocks.
+	// A new template part is automatically created if we have any inner blocks but no entity.
```

Comment:
```suggestion
	// If we have a reference to the Navigation Post but the post entity no longer exists
	// then show a warning to the user.
```

Question: if the users ends up in this state can we provide them with an option to create a new navigation rather than having to remove the block and re-add it?